assets/js/components/Role/NavDropdown.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useContext } from "react";
+import { useMutation } from "@apollo/client";
+import { AuthContext } from "@js/components/Auth/Auth";
+import UILayoutNavDropdown from "@js/components/UI/Layout/NavDropdown";
+import UILayoutNavDropdownHeader from "@js/components/UI/Layout//NavDropdownHeader";
+import UILayoutNavDropdownBody from "@js/components/UI/Layout/NavDropdownBody";
+import UILayoutNavDropdownItem from "@js/components/UI/Layout/NavDropdownItem";
+import { ASSUME_ROLE } from "@js/components/Role/role.gql.js";
+import { GET_CURRENT_USER_QUERY } from "@js/components/Auth/auth.gql";
+import { toastWrapper } from "@js/services/helpers";
+
+const RoleNavDropdown = () => {
+  const currentUser = useContext(AuthContext);
+  const [activeHoverNav, setActiveHoverNav] = useState("adam");
```

Comment:
Did I accidentally leave this in previously?  Hopefully code is not cueing off of  a state value of `adam`?

Reply:
lol. yeah I copied and pasted it from here. https://github.com/nulib/meadow/blob/0fd27eb547d897d59e9dd4294bd53f2843f19ec1/assets/js/components/UI/Layout/NavBar.jsx#L24 I wasn't sure of the secret meaning. 

But yeah, I think I could make it a submenu of user. I think that' what you're saying. I'll look at the Bulma docs a bit.

Reply:
I'll remove both "adam"s

Reply:
And, not to create more work, but looks like Bulma's CSS and markup for displaying a nav dropdown item and list might have changed slightly from when we implemented it?  (In case things don't seem to display quite right)